installer/ApplicationSetupUtility.php

```diff
@@ -32,7 +32,7 @@ class ApplicationSetupUtility {
      * @return string
      */
     public static function getErrorLogPath() {
```

Comment:
Web installer working fine since logfile creating another place.
But CLI installer failing since realpath returning false (file not created).
Need to adjest bit;
```php
    public static function getErrorLogPath() {
        return realpath(__DIR__ . '/../symfony/log') . DIRECTORY_SEPARATOR.  'installer.log';
    }
```